packages/starlight/__e2e__/fixtures/ssr/src/content/docs/demo.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Tabs
```

Comment:
We should probably change the title of this page and also remove the content of this page involving `Tabs` as it's a duplicate of another set of tests which should not be needed here.